dspace-api/src/test/java/org/dspace/app/util/SubmissionConfigTest.java

```diff
@@ -87,4 +109,35 @@ public void testReadAndProcessTypeBindSubmissionConfig()
         assertEquals(unboundFields, allowedFieldsForArticle);
         assertEquals(unboundFields, allowedFieldsForNoType);
     }
+
+    @Test
+    public void testSubmissionMapByCommunityHandleSubmissionConfig()
+        throws SubmissionConfigReaderException, DCInputsReaderException, SQLException {
+
+        // Sep up a structure with one top community and two subcommunities
+        // with one collection
+        when(col1.getHandle()).thenReturn("123456789/not-mapped1");
+        when(col1.getCommunities()).thenReturn(List.of(subcom1));
+
+        when(col2.getHandle()).thenReturn("123456789/not-mapped2");
+        when(col2.getCommunities()).thenReturn(List.of(subcom2));
+
+        when(subcom1.getHandle()).thenReturn("123456789/subcommunity-test");
```

Comment:
subcom1 should be a child of topcom to verify that the configuration of the "close" community is used